### ISSUE TITLE
Question: Commit id and hyper-parameter settings for results in "benchmarks_atari10M.htm"

### ISSUE THREAD

**Issue by whyjay** [openai/baselines#522]
Appreciate you for serving with this great baselines. 
I am running ACER in this commit [ea68f3b](https://github.com/openai/baselines/commit/ea68f3b7e6a20d4c6bf1e32f8fb5ce18e6ef3a89) with default hyper-parameter settings. 
Since i found this commit in `benchmarks_atari10M.htm`, it should reproduce the benchmark results. But it seems to learn significantly slower (or maybe not as good as the reported benchmark results)
I use Ubuntu 16.04.4, Python 3.5.2 and tensorflow 1.10.

My Questions:
1) Did i choose the right commit which reproduce the benchmark results?
2) Possibly, your hyper-parameter settings were not default ones. If so, please let me know of hyper-parameter settings that you used for the PPO2/A2C/ACER/DeepQ benchmark results.
3) Is there any other changes i should make?

Followings are tabular stdout logs for 7 tasks at near 5 million step.
Thank you!

BeamRider
----------------------------------
| avg_norm_adj        | 0.0485   |
| avg_norm_g          | 1.49     |
| avg_norm_grads_f    | 1.46     |
| avg_norm_k          | 3.01     |
| avg_norm_k_dot_g    | 1.47     |
| entropy             | 2.12     |
| explained_variance  | 0.807    |
| fps                 | 722      |
| loss                | -0.0654  |
| loss_bc             | -0       |
| loss_f              | -0.0615  |
| loss_policy         | -0.0615  |
| loss_q              | 0.0347   |
| mean_episode_length | 482      |
| mean_episode_reward | 4.92     |
| norm_grads          | 0.15     |
| norm_grads_policy   | 0.101    |
| norm_grads_q        | 0.118    |
| total_timesteps     | 5040000  |
----------------------------------

Breakout
----------------------------------
| avg_norm_adj        | 0.0784   |
| avg_norm_g          | 1.11     |
| avg_norm_grads_f    | 1.06     |
| avg_norm_k          | 2.09     |
| avg_norm_k_dot_g    | 1.06     |
| entropy             | 1.09     |
| explained_variance  | 0.593    |
| fps                 | 756      |
| loss                | -0.0216  |
| loss_bc             | -0       |
| loss_f              | -0.0532  |
| loss_policy         | -0.0532  |
| loss_q              | 0.085    |
| mean_episode_length | 144      |
| mean_episode_reward | 3.45     |
| norm_grads          | 0.397    |
| norm_grads_policy   | 0.262    |
| norm_grads_q        | 0.268    |
| total_timesteps     | 5040000  |
----------------------------------

Enduro
-----------------------------------
| avg_norm_adj        | 0         |
| avg_norm_g          | 0.139     |
| avg_norm_grads_f    | 0.139     |
| avg_norm_k          | 3         |
| avg_norm_k_dot_g    | 0.165     |
| entropy             | 2.2       |
| explained_variance  | -0.137    |
| fps                 | 707       |
| loss                | -0.0222   |
| loss_bc             | -0        |
| loss_f              | -0.000368 |
| loss_policy         | -0.000368 |
| loss_q              | 0.000222  |
| mean_episode_length | 3.32e+03  |
| mean_episode_reward | 0         |
| norm_grads          | 0.00106   |
| norm_grads_policy   | 0.000819  |
| norm_grads_q        | 0.00068   |
| total_timesteps     | 5040000   |
-----------------------------------

Pong
----------------------------------
| avg_norm_adj        | 0.158    |
| avg_norm_g          | 1.77     |
| avg_norm_grads_f    | 1.69     |
| avg_norm_k          | 2.57     |
| avg_norm_k_dot_g    | 1.6      |
| entropy             | 1.64     |
| explained_variance  | 0.77     |
| fps                 | 801      |
| loss                | -0.136   |
| loss_bc             | -0       |
| loss_f              | -0.137   |
| loss_policy         | -0.137   |
| loss_q              | 0.0344   |
| mean_episode_length | 2.17e+03 |
| mean_episode_reward | -12.7    |
| norm_grads          | 0.335    |
| norm_grads_policy   | 0.181    |
| norm_grads_q        | 0.259    |
| total_timesteps     | 4960000  |
----------------------------------

Qbert
----------------------------------
| avg_norm_adj        | 0.204    |
| avg_norm_g          | 6.76     |
| avg_norm_grads_f    | 6.63     |
| avg_norm_k          | 2.55     |
| avg_norm_k_dot_g    | 6.38     |
| entropy             | 1.17     |
| explained_variance  | 0.928    |
| fps                 | 708      |
| loss                | 0.785    |
| loss_bc             | -0       |
| loss_f              | -0.0246  |
| loss_policy         | -0.0246  |
| loss_q              | 1.64     |
| mean_episode_length | 198      |
| mean_episode_reward | 16       |
| norm_grads          | 3.63     |
| norm_grads_policy   | 0.752    |
| norm_grads_q        | 3.67     |
| total_timesteps     | 5040000  |
----------------------------------

Seaquest
----------------------------------
| avg_norm_adj        | 0.0904   |
| avg_norm_g          | 4.05     |
| avg_norm_grads_f    | 4        |
| avg_norm_k          | 4.28     |
| avg_norm_k_dot_g    | 4.03     |
| entropy             | 2.76     |
| explained_variance  | 0.764    |
| fps                 | 724      |
| loss                | -0.0698  |
| loss_bc             | -0       |
| loss_f              | -0.0695  |
| loss_policy         | -0.0695  |
| loss_q              | 0.0546   |
| mean_episode_length | 469      |
| mean_episode_reward | 8.55     |
| norm_grads          | 0.146    |
| norm_grads_policy   | 0.0924   |
| norm_grads_q        | 0.121    |
| total_timesteps     | 5040000  |
----------------------------------

SpaceInvaders
----------------------------------
| avg_norm_adj        | 0.112    |
| avg_norm_g          | 2.45     |
| avg_norm_grads_f    | 2.36     |
| avg_norm_k          | 2.76     |
| avg_norm_k_dot_g    | 2.71     |
| entropy             | 1.51     |
| explained_variance  | 0.886    |
| fps                 | 715      |
| loss                | 0.0998   |
| loss_bc             | -0       |
| loss_f              | 0.00865  |
| loss_policy         | 0.00865  |
| loss_q              | 0.212    |
| mean_episode_length | 330      |
| mean_episode_reward | 12       |
| norm_grads          | 0.656    |
| norm_grads_policy   | 0.414    |
| norm_grads_q        | 0.492    |
| total_timesteps     | 5040000  |
----------------------------------

**Comment by whyjay** [comment]
I looked into generated `monitor.csv` files, and

**Comment by KhuongDiep911** [comment]
Hi whyjay,

Are you till using this baselines? Can you share with me your environment?
Because I need to use the tensorflow 1.10. But I can not run the baselines on gpu. I really appreciate if you can help me.

Thank you in advance, 
Khuong
